File: app/src/main/java/com/mrbattery/stratus/candybar/activities/MuzeiActivity.java
package com.mrbattery.stratus.candybar.activities;

import android.support.annotation.NonNull;

import com.dm.material.dashboard.candybar.activities.CandyBarMuzeiActivity;
import com.mrbattery.stratus.candybar.services.MuzeiService;

public class MuzeiActivity extends CandyBarMuzeiActivity {

    @NonNull
    @Override
    public Class<?> onInit() {
        return MuzeiService.class;
    }
}

File: app/src/main/java/com/mrbattery/stratus/candybar/activities/MuzeiActivity.kt
package com.mrbattery.stratus.candybar.activities

import com.dm.material.dashboard.candybar.activities.CandyBarMuzeiActivity
import com.mrbattery.stratus.candybar.services.MuzeiService

class MuzeiActivity : CandyBarMuzeiActivity() {
    override fun onInit(): Class<*> {
        return MuzeiService::class.java
    }
}